Makefile

```diff
@@ -292,27 +311,47 @@ clean-coverage:
 
 # Building Docker Images for our executables
 ############################################
-images: user-broker-image \
-    controller-manager-image apiserver-image
+images: user-broker-image controller-manager-image apiserver-image
+
+images-all: $(addprefix sub-images-,$(ALL_ARCH))
+sub-images-%:
```

Comment:
So, the `images-all` target runs `sub-image-$EACH_ARCH`, right?  Would it be better to call it `arch-image-`?